tensorboard/webapp/plugins/npmi/views/annotations_list/legend/legend_component.ng.html

```diff
@@ -0,0 +1,40 @@
+<!--
+@license
+Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<npmi-legend-element
```

Comment:
I need a little more background before doing a proper review: Do you expect the legend will always consist of a small number of hardcoded elements?  Do you foresee any more elements being added by the end of your internship or by the end of the year? If so, which ones and which colors/shapes would they have?

Reply:
Yes, it is normally a small number of fixed elements, and for now, I dont plan to add more.